src/binary/ibc-setup/commands/init.ts

```diff
@@ -46,17 +88,42 @@ export async function run(options: Options) {
       const registryFromRemote = await axios.get(
         'https://raw.githubusercontent.com/confio/ts-relayer/main/demo/registry.yaml'
       );
-      fs.writeFileSync(registryFilePath, registryFromRemote.data, {
-        encoding: 'utf-8',
-      });
+      fs.writeFileSync(registryFilePath, registryFromRemote.data);
     } catch (error) {
       throw new Error(`Cannot fetch ${registryFile} from remote. ${error}`);
     }
   } else if (!fs.lstatSync(registryFilePath).isFile()) {
     throw new Error(`${registryFilePath} must be a file.`);
   }
 
-  const registry = yaml.load(fs.readFileSync(registryFilePath, 'utf-8'));
+  const registry = yaml.load(
+    fs.readFileSync(registryFilePath, 'utf-8')
+  ) as Registry;
   // TODO #75: registry file validation
-  console.log(registry);
+
+  const chainSrc = registry.chains[options.src];
+  const chainDest = registry.chains[options.dest];
+
+  const mnemonic = generateMnemonic();
+
+  const appYaml = yaml.dump(
+    {
+      src: options.src,
+      dest: options.dest,
+      mnemonic,
+    },
+    {
+      lineWidth: 1000,
```

Comment:
Ah, nice to avoid the break. 1000 = never line break

Can you add a comment ("to ensure mnemonic is not split on multiple lines")

Reply:
117f41f